packages/database/src/migrations/20210202031622-NutritionCounsellingTotalNewAndExistingClients-modifies-data.js

```diff
@@ -0,0 +1,286 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+async function addReportToGroups(db, reportId, groupCodes) {
+  return db.runSql(`
+    UPDATE
+      "dashboardGroup"
+    SET
+      "dashboardReports" = "dashboardReports" || '{"${reportId}"}'
+    WHERE
+      "code" IN (${groupCodes.map(code => `'${code}'`).join(',')});
+  `);
+}
+async function removeReportFromGroups(db, reportId, groupCodes) {
+  return db.runSql(`
+    UPDATE
+      "dashboardGroup"
+    SET
+      "dashboardReports" = array_remove("dashboardReports", '${reportId}')
+    WHERE
+      "code" IN (${groupCodes.map(code => `'${code}'`).join(',')});
+  `);
+}
+async function deleteItemByOneCondition(db, table, condition) {
+  const [key, value] = Object.entries(condition)[0];
+  return db.runSql(`
+    DELETE FROM
+      "${table}"
+    WHERE
+      "${key}" = '${value}';
+  `);
+}
+const totalClientNumberIndicator = {
+  id: generateId(),
+  code: 'TO_HPU_Nutrition_Counselling_Number_Of_Total_Client',
+  builder: 'arithmetic',
+  config: {
+    formula:
+      'HP299b + HP299c + HP300a + HP300b + HP301a + HP301b + HP302a_m + HP302a_f +HP302b_m + HP302b_f + HP303 + HP304 + HP323aa + HP323ab + HP324a + HP324b + HP325a + HP325b + HP326a_m + HP326a_f + HP326b_m + HP326b_f + HP327 + HP328',
+    aggregation: 'SUM_EACH_MONTH',
+    defaultValues: {
+      HP299b: 0,
+      HP299c: 0,
```

Comment:
It seems that it would be handy to allow a single number as a default value for all variables in the formula:

`defaultValues: 0`

A feature to keep in mind for the future of indicators 😃

Reply:
That would be nice 👍  No need to proofread them with the formula.